README.md

```diff
@@ -595,7 +595,9 @@ Options:
 * Probably only works on unix/bash for now (PRs welcome)
   * Should probably run the build in a docker container anyway
 * `yarn install`
-* `yarn build`
+* `MAPBOX_ACCESS_TOKEN=your-token-here yarn build`
+  * Please note that you will need a Mapbox token in order to successfully run `yarn build`, otherwise the build will fail due to failing to load the map images. You can get your own token [here](https://account.mapbox.com/access-tokens). Once you have a token, you should assign it to the `MAPBOX_ACCESS_TOKEN` environment variable.
+
```

Comment:
Does this extra newline separate out things into two lists?

Reply:
oops